src/components/Form/FormWrapper.tsx

```diff
@@ -104,7 +104,7 @@ function FormWrapper({
                 ref={formContentRef}
                 style={[style, safeAreaPaddingBottomStyle.paddingBottom ? safeAreaPaddingBottomStyle : styles.pb5]}
             >
-                {children}
+                <View style={styles.flex1}>{children}</View>
```

Comment:
Did you test that this works with all existing forms in the app? Or is it unlikely to break anything?